src/containers.ts

```diff
@@ -25,7 +25,7 @@ export type HelperFunction = (
   config: SqrlConfig
 ) => string | Promise<string>
 
-export type FilterFunction = (str: string) => any | Promise<any>
+export type FilterFunction = (...args: string[]) => any | Promise<any>
```

Comment:
Will you change the signature to something like `(val: any, ...filterParams: any[])`?

A filter function should probably always take at least the first parameter. Also, filters can actually accept more than just strings. For example, `{{ it.arr | join(",") | truncate(3) }}` would be completely valid syntax.

Thanks again!

Reply:
I will make that change now! was just following with